source/index.html

```diff
@@ -2,11 +2,164 @@
+            <li class="catalogs__item">
+              <div class="catalogs__item-box">
+                <div class="catalogs__item-header">
+                  <img class="catalogs__item-icon" src="" alt="Иконка каталога">
+                  <h2 class="catalogs__item-title">Набор массы</h2>
+                </div>
+                <p class="catalogs__item-text">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+                <a href="#" class="catalogs__item-link">каталог pro</a>
+              </div>
+              <img class="catalogs__item-icon catalogs__item-logo--table" src="" alt="Логотип каталога">
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="howitwork">
+        <div class="container">
+          <h2 class="howitwork__title">Как это работает</h2>
+          <ul class="howitwork__list">
+            <li class="howitwork__item">
+              <img src="" alt="Иконка пункта" class="howitwork__item-icon">
+              <p class="howitwork__item-text">Функциональное питание содержит только полезные питательные вещества.</p>
+              <h3 class="howitwork__item-number">1</h3>
+            </li>
+            <li class="howitwork__item">
+              <img src="" alt="Иконка пункта" class="howitwork__item-icon">
+              <p class="howitwork__item-text">Выпускается в виде порошка, который нужно лишь залить кипятком и готово.</p>
+              <h3 class="howitwork__item-number">2</h3>
+            </li>
+            <li class="howitwork__item">
+              <img src="" alt="Иконка пункта" class="howitwork__item-icon">
+              <p class="howitwork__item-text">Замените один-два приема обычной еды на наше функциональное питание.</p>
+              <h3 class="howitwork__item-number">3</h3>
+            </li>
+            <li class="howitwork__item">
+              <img src="" alt="Иконка пункта" class="howitwork__item-icon">
+              <p class="howitwork__item-text">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
+              <h3 class="howitwork__item-number">4</h3>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="example">
+        <div class="container">
+          <h2 class="example__title">Живой пример</h2>
+          <div class="example__lore">
+            <p class="example__lore-text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+            <dl class="example__lore-info">
+              <dt class="example__info-title">5 КГ</dt>
+              <dd class="example__info-desc">снижение веса</dd>
+              <dt class="example__info-title">60 ДНЕЙ</dt>
+              <dd class="example__info-desc">затрачено времени</dd>
+            </dl>
```

Comment:
Текст надо записывать в естественном виде, потом стилями приводить к нужному виду. И вместо desc лучше писать description, от таких сокращений нет никакой пользы. 
Ну и вообще, здесь наверное не стоит использовать список определений, либо разметить его по другому немного
```suggestion
            <dl class="example__lore-info">
              <div>
                <dt class="example__info-title">5 кг</dt>
                <dd class="example__info-description">Снижение веса</dd>
              </div>
              <div>
                <dt class="example__info-title">60 дней</dt>
                <dd class="example__info-description">Затрачено времени</dd>
              </div>
            </dl>
```